terraform/modules/scheduled-job/README.md

```diff
@@ -389,6 +391,42 @@ Or use Cloud Build directly:
 gcloud builds submit --tag gcr.io/YOUR_PROJECT_ID/YOUR_JOB_NAME:latest ./jobs/your-job
 ```
 
+## Resource Tagging
+
+All resources created by this module are automatically tagged with common metadata:
+
+### Automatic Tags
+- `terraform_module` - Set to "scheduled-job"
+- `scheduled_job_name` - The name of your function/job
+- `owner` - The owner/team responsible for this scheduled job
+
+### Custom Tags
+You can add custom tags using the `tags` variable:
+
+```hcl
+module "my_function" {
+  source = "git::https://github.com/Khan/terraform-modules.git//terraform/modules/scheduled-job?ref=v1.0.0"
+  
+  job_name = "my-function"
+  owner   = "data-team"
+  # ... other configuration
+  
+  tags = {
+    "environment" = "production"
+    "team"        = "data-engineering"
+    "cost-center" = "infrastructure"
+  }
+}
+```
+
+### Supported Resources
+The following resources support tagging/labeling:
```

Comment:
What happens if you try to add tags somewhere else?  It seems like it's just silently ignored?